# get a list of frontier centriods

#map
nav_msgs/OccupancyGrid map
---
# list of frontier centroids
geometry_msgs/Point[] centroids
